src/commonTest/kotlin/expressions/VarTest.kt

```diff
@@ -0,0 +1,90 @@
+package expressions
+
+import JsonLogicEngine
+import TestInput
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+
+class VarTest : FunSpec({
+    context("JsonLogic evaluation with Var operation") {
+        withData(
+            // given
+            listOf(
+                /* Unsupported test cases
```

Comment:
maybe create separate test file for this later?

Reply:
Moved these cases to some val under the testing class body to keep it in one place and not cover up the active code